src/components/TransactionItemRow/DataCells/ChatBubbleCell.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import {ChatBubble} from '@components/Icon/Expensicons';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {getIOUActionForReportID, getReportAction} from '@libs/ReportActionsUtils';
+import {getReportOrDraftReport, isChatThread} from '@libs/ReportUtils';
+import type Transaction from '@src/types/onyx/Transaction';
+
+type MessagesCountAndStatus = {
+    reportID: string | undefined;
+    count: number;
+    isUnread: boolean;
+};
+
+function getTransactionMessagesCountAndUnreadInfo(transaction: Transaction) {
+    const result: MessagesCountAndStatus = {
+        reportID: undefined,
+        count: 0,
+        isUnread: false,
+    };
+
+    const iouReport = getIOUActionForReportID(transaction.reportID, transaction.transactionID);
+
+    if (!iouReport) {
+        return result;
+    }
+
+    const count = iouReport.childVisibleActionCount;
+    const action = getReportAction(transaction.reportID, iouReport.reportActionID);
+
+    if (!action) {
+        return result;
+    }
+
+    result.reportID = action.reportID;
```

Comment:
Could you add plain english comments to the steps in this method so its easier to follow the logic, thanks!